text/stacks/0007-squash-stack.md

```diff
@@ -0,0 +1,45 @@
+# Squash Stack Images
+
+## Summary
```

Comment:
Are there any concerns with the fact that a single layer will effectively disable any caching? I'm assuming not, given that the bulk of the stacks (i.e. the package installation) is currently all in a single layer and the layers above are minimal in size.

Reply:
I think users already experience little in the way of caching since the package layer is just one big layer and we pretty much change it on every release. Technically, yes, this makes caching the stack worse. But in reality, I believe that most users benefited very little from that caching to begin with.

Reply:
Yeah, that's pretty much what I thought, too. I just wanted to be explicit about that consequences of this decision; why/how we are ok with the lack of caching.